Kernel/Time/TimeManager.cpp

```diff
@@ -27,6 +32,13 @@ KErrorOr<timespec> TimeManager::get_time(int clock_id)
         ts.tv_nsec = miliseconds_since_boot * 1000000;
         return ts;
     }
+    
+    if (clock_id == CLOCK_REALTIME) {
```

Comment:
Now It seems like CLOCK_REALTIME is the same as CLOCK_MONOTONIC_COARSE.
According to the [specification](https://linux.die.net/man/2/clock_gettime), CLOCK_REALTIME should return "real" time but not a time since boot.